dallinger/data.py

```diff
@@ -265,9 +270,13 @@ def export_db_uri(id, db_uri, local, scrub_pii):
 
     # Backup data on S3 unless run locally
     if not local:
-        bucket = user_s3_bucket()
         config = get_config()
-        try:
+
+        bucket = None
+        if aws_access_keys_present(config):
+            bucket = user_s3_bucket()
+
+        if bucket is not None:
```

Comment:
Why are you testing if the bucket is None? None is no longer an output from `user_s3_bucket`. If you are testing rather whether `aws_access_keys_present` returns `True`, you should make that more explicit.

Reply:
Addressed this one, too.